trace/report_buffer.go

```diff
@@ -39,6 +42,8 @@ func (buf *reportBuffer) Add(raw rawReport) (bool, error) {
 		witnessReport, err := raw.Witness.toReport()
 		if err != nil {
 			printer.Warningf("Failed to convert witness to report: %v\n", err)
+		} else if maxSize, exists := buf.maxWitnessSize_bytes.Get(); exists && len(witnessReport.WitnessProto) > maxSize {
+			// Drop the witness; it's too large.
```

Comment:
Should at least have a debug statement here; you've already discussed adding a count to packet telemetry.
(Which might be _more_ difficult if the BackendCollector and its reportBuffer is not associated with a particular interface any more.)

Reply:
Added debug message in 65fa486.